Connected web3 provider at 10.195.2.242:22008
Date.now(): 1579050181087 startTime: 1579050286274 delay: 105190
Deploying smart contract from 0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a
TX HASH:
 0x3c0c10612f58b711a718f2f8869ad879e06a3ca0087dbc627d35bde86a05929a
RECEIPT:
 {
  blockHash: '0x9f96a557994dbe9172a2399d2557528d2c98d8191a68987bb7862a7f9bb84578',
  blockNumber: 7,
  contractAddress: '0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7',
  cumulativeGasUsed: 218495,
  from: '0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0x3c0c10612f58b711a718f2f8869ad879e06a3ca0087dbc627d35bde86a05929a',
  transactionIndex: 0
}
Contract at: 0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7

Evaluating  6400txs...

Sending TXS started at 1579050286373 ...
startingBlock: 20
... Sending TXS finished at 1579050387399
Total send duration: 101026
... Waiting for mining process to finish! ...
(node:4355) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
(node:4355) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4355) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
